src/PASopa/Program.cs

```diff
@@ -34,6 +34,48 @@ static void Main(string[] args)
                 MsAppTest.StressTest(msAppPath);
                 return;
             }
+
+
+            if (mode == "-testall2")
+            {
+                if (args.Length < 2)
+                {
+                    Usage();
+                    return;
+                }
+                // Roundtrip all .msapps in a folder.
+                string msAppPathDir = args[1];
+                int countTotal = 0;
+                int countPass = 0;
+                Console.WriteLine("Test2 roundtripping all .msapps in : " + msAppPathDir);
+                string msAppCommon = null;
+                foreach (var msAppPath in Directory.EnumerateFiles(msAppPathDir, "*.msapp", SearchOption.TopDirectoryOnly))
+                {
+                    // Merge test requires a 2nd app. Could do a full NxN matrix. But here, just pick the first item. 
+                    msAppCommon ??= msAppPath;
+
+                    Stopwatch sw = Stopwatch.StartNew();
+                    // bool ok = MsAppTest.DiffStressTest(msAppPath); // all pass!
+                    //bool ok = true;
+                    bool ok = MsAppTest.TestClone(msAppPath); 
```

Comment:
Is this supposed to be testing clone, or merge?

Reply:
I'll remove this code and just roll TestClone into the existin TestAll